src/shared/resourcefetcher/httpResourceFetcher.ts

```diff
@@ -148,6 +183,22 @@ export class HttpResourceFetcher implements ResourceFetcher {
 
         return promise
     }
+
+    private buildRequestHeaders(requestHeaders?: RequestHeaders): Headers {
+        const headers: Headers = {}
+
+        headers['User-Agent'] = VSCODE_EXTENSION_ID.awstoolkit
+
+        if (requestHeaders?.eTag !== undefined) {
+            headers['If-None-Match'] = requestHeaders.eTag
+        }
+
+        if (requestHeaders?.gZip) {
+            headers['Accept-Encoding'] = 'gzip'
```

Comment:
Where does decompression happen?

Reply:
I was assuming I had to decompress the result myself at some point, but I think the http request module is doing it for me.

I verified and the Result object has the header `Content-Encoding: gzip`, but the text given to me is uncompressed.
---
I tested with `wget` + `Accept-Encoding: gzip` and that result is compressed, so the compression is working as expected.

Reply:
Let's at least leave a comment somewhere here, mentioning that info

Reply:
Added a comment to the line where we read the data, indicating we don't need to decompress the data.